annsrcs/ensembl/oryza_glumaepatula

```diff
@@ -3,7 +3,7 @@ software.version=39
 url=http://plants.ensembl.org/biomart/martservice?
 organism=oryza glumaepatula
 databaseName=plants
-datasetName=oglumaepatula_eg_gene
+datasetName=oglumipatula_eg_gene
```

Comment:
Not sure why BioMart has used two different names Oryza glumipatula and Oryza_glumaepatula which is inconsistent. However, thanks for the correction.